packages/common/src/GRPCClient.ts

```diff
@@ -557,6 +557,87 @@ export default class ConcordiumNodeClient {
             .response;
         return translate.unwrapValToHex(response);
     }
+
+    /**
+     * Get the identity providers registered as of the end of a given block.
+     * The stream will end when all the identity providers have been returned,
+     * or an abort signal is called.
+     *
+     * @param blockHash an optional block hash to get the instance states at, otherwise retrieves from last finalized block.
+     * @param abortSignal an optional AbortSignal to close the stream.
+     * @returns an async iterable of identity provider info objects.
+     */
+    getIdentityProviders(
+        blockHash?: HexString,
+        abortSignal?: AbortSignal
+    ): AsyncIterable<v1.IpInfo> {
+        const opts = { abort: abortSignal };
+        const block = getBlockHashInput(blockHash);
+        const ips = this.client.getIdentityProviders(block, opts).responses;
+        return mapAsyncIterable(ips, translate.ipInfo);
+    }
+
+    /**
+     * Get the anonymity revokers registered as of the end of a given block.
+     * The stream will end when all the anonymity revokers have been returned,
+     * or an abort signal is called.
+     *
+     * @param blockHash an optional block hash to get the instance states at, otherwise retrieves from last finalized block.
```

Comment:
```suggestion
     * @param blockHash an optional block hash to get the anonymity revokers states at, otherwise retrieves from last finalized block.
```